src/main/java/com/ericsson/gerrit/plugins/eiffel/state/SourceChangeCreatedState.java

```diff
@@ -0,0 +1,33 @@
+package com.ericsson.gerrit.plugins.eiffel.state;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.net.ConnectException;
+import java.sql.SQLException;
+
+import com.ericsson.gerrit.plugins.eiffel.events.EiffelEvent;
+import com.ericsson.gerrit.plugins.eiffel.events.EiffelSourceChangeCreatedEvent;
+import com.ericsson.gerrit.plugins.eiffel.handlers.NoSuchElementException;
+import com.ericsson.gerrit.plugins.eiffel.handlers.Table;
+
+public class SourceChangeCreatedState extends State {
+
+    public SourceChangeCreatedState(File pluginDir) {
```

Comment:
This PR adds States for each event type, and StateAccesors for each event type, if there will ever be more than 2 event types this will become many many files. It also seems like there is duplicated code on most of the files. This could be done better, maybe there should be a team discussion about this.